packages/cli/lib/projects.js

```diff
@@ -618,6 +616,32 @@ const logFeedbackMessage = buildId => {
   }
 };
 
+const createProjectComponent = async (component, name) => {
+  const i18nKey = 'cli.commands.project.subcommands.add';
+  let componentName = name;
+
+  const directoryPath = path.resolve(getCwd());
+  const configInfo = await getProjectConfig(directoryPath);
```

Comment:
You don't need to pass `directoryPath` in here. This util will use `CWD` by default if no path is passed in.